### ISSUE TITLE
2021-10-2

### ISSUE THREAD

**Issue by ssfc** [ssfc/wangdao#619]
511I.Introduction
5121.“distillation” to transfer the knowledge from the cumbersome model to a small model that is more suitable for deployment; (2021-9-27)
5132.we tend to identify the knowledge in a trained model with the learned parameter values and this makes it hard to see how we can change the form of the model but keep the same knowledge; relative probabilities of incorrect answers tell us a lot about how the cumbersome model tends to generalize; BMW == truck != bird; (2021-9-28)
5143.objective function used for training should reflect the true objective of the user as closely as possible; better to train models to generalize well; a small model trained to generalize in the same way will typically do much better on test data than a small model that is trained in the normal way on the same training set as was used to train the ensemble; (2021-9-28)
5154.An obvious way to transfer the generalization ability of the cumbersome model to a small model is to use the class probabilities produced by the cumbersome model as “soft targets” for training the small model; use an arithmetic or geometric mean of their individual predictive distributions as the soft targets; When the soft targets have high entropy, they providemuch more information per training case than hard targets; small model can often be trained on much less data than the original cumbersome model; (2021-9-28)
5165.it says which 2’s look like 3’s and which look like 7’s; raise the temperature (T) of the final softmax until the cumbersome model produces a suitably soft set of targets; then use the same high temperature when training the small model to match these soft targets; (2021-9-28)
5176.we add a small term to the objective function that encourages the small model to predict the true targets as well as matching the soft targets provided by the cumbersome model; (2021-9-29)
518II.Distillation
5191.Using a higher value for T produces a softer probability distribution over classes; (2021-9-29)
